core/src/test/java/com/taobao/arthas/core/util/StringUtilsTest.java

```diff
@@ -18,4 +27,388 @@ public void testHumanReadableByteCount() {
+
+        Assert.assertNull(StringUtils.toS   tringArray(collection));
+        Assert.assertArrayEquals(new String[] {}, StringUtils.toStringArray(arrayList));
+        Assert.assertArrayEquals(new String[] {"foo"}, StringUtils.toStringArray(arrayList2));
+    }
+
+    @Test
+    public void testTrimAllWhitespace() {
+        Assert.assertEquals("", StringUtils.trimAllWhitespace(" "));
+        Assert.assertEquals("", StringUtils.trimAllWhitespace(""));
+        Assert.assertEquals("foo", StringUtils.trimAllWhitespace("foo"));
+        Assert.assertEquals("foo", StringUtils.trimAllWhitespace("  foo  "));
+    }
+
+    @Test
+    public void testTrimLeadingCharacter() {
+        Assert.assertEquals("", StringUtils.trimLeadingCharacter("", 'a'));
+        Assert.assertEquals("foo", StringUtils.trimLeadingCharacter("foo", 'a'));
+        Assert.assertEquals("", StringUtils.trimLeadingCharacter("a", 'a'));
+        Assert.assertEquals("foo", StringUtils.trimLeadingCharacter("afoo", 'a'));
+    }
+
+    @Test
+    public void testTrimLeadingWhitespace() {
+        Assert.assertEquals("", StringUtils.trimLeadingWhitespace(""));
+        Assert.assertEquals("", StringUtils.trimLeadingWhitespace(" "));
+        Assert.assertEquals("!", StringUtils.trimLeadingWhitespace("!"));
+        Assert.assertEquals("foo  ", StringUtils.trimLeadingWhitespace("  foo  "));
+    }
+
+    @Test
+    public void testTrimTrailingCharacter() {
+        Assert.assertEquals("foo", StringUtils.trimTrailingCharacter("foo!", '!'));
+        Assert.assertEquals("", StringUtils.trimTrailingCharacter("", '!'));
+        Assert.assertEquals("", StringUtils.trimTrailingCharacter("!", '!'));
+    }
+
+    @Test
+    public void testTrimTrailingWhitespace() {
+        Assert.assertEquals("", StringUtils.trimTrailingWhitespace(" "));
+        Assert.assertEquals("", StringUtils.trimTrailingWhitespace(""));
+        Assert.assertEquals("  foo", StringUtils.trimTrailingWhitespace("  foo   "));
+    }
+
+    @Test
+    public void testTrimWhitespace() {
+        Assert.assertEquals("\u6d7c\u1280", StringUtils.trimWhitespace(" \u6d7c\u1280\u1680"));
+        Assert.assertEquals("", StringUtils.trimWhitespace(" "));
+        Assert.assertEquals("(", StringUtils.trimWhitespace("("));
+        Assert.assertEquals("", StringUtils.trimWhitespace(""));
+        Assert.assertEquals("foo foo", StringUtils.trimWhitespace(" foo foo "));
+    }
+
+    @Test
+    public void testUncapitalize() {
+        Assert.assertEquals("a", StringUtils.uncapitalize("A"));
+        Assert.assertEquals("a", StringUtils.uncapitalize("a"));
+        Assert.assertEquals("", StringUtils.uncapitalize(""));
+    }
+
+    @Test
+    public void testCapitalize() {
+        Assert.assertEquals("A", StringUtils.capitalize("a"));
+        Assert.assertEquals("A", StringUtils.capitalize("A"));
+        Assert.assertEquals("", StringUtils.capitalize(""));
+    }
+
+    @Test
+    public void testUnqualify() {
```

Comment:
What is the context of this method? Does it only remove `.` or also do `a.b.c` -> `c`, for instance ?